apps/module-scan/src/types.ts

```diff
@@ -46,11 +46,20 @@ export function getMarkStatus(
     return MarkStatus.Marked
   }
 
-  if (mark.score < markThresholds.marginal) {
-    return MarkStatus.Unmarked
+  if (mark.score >= markThresholds.marginal) {
+    return MarkStatus.Marginal
   }
 
-  return MarkStatus.Marginal
+  if (
+    mark.type === 'candidate' &&
+    typeof mark.writeInTextScore === 'number' &&
+    typeof markThresholds.writeInText === 'number' &&
+    mark.writeInTextScore >= markThresholds.writeInText
```

Comment:
is there some additional condition here that says "this is a mark related to a write-in?

or is that part of the other conditions?